docs/how-to/faq.md

```diff
@@ -283,6 +283,14 @@ One notable exception is that `hipError_t` is a new type, and cannot be used whe
 
 If platform portability is important, use `#ifdef __HIP_PLATFORM_NVIDIA__` to guard the CUDA-specific code.
 
+## Why can't NVCC compile my .hip files?
+
+NVCC does not recognize files with the `.hip` file extension. You must rename the source file extension to one NVCC supports, like `.cpp`. See [NVCC Supported Input File suffixes](https://docs.nvidia.com/cuda/cuda-compiler-driver-nvcc/#supported-input-file-suffixes).
```

Comment:
I wouldn't immediately tell the user to rename their files as a solution to this.

- HIP specifically provides hipcc as a wrapper around nvcc and amdclang++ to avoid these problems, so using hipcc would be one solution to this (even if that might not be desired by everyone, as hipcc also has some weird default settings)
- If that's not desired, the file extension does not have to be renamed, that sounds more like a last resort. nvcc provides an option like almost every other compiler to specify the language of the file to be compiled (https://docs.nvidia.com/cuda/cuda-compiler-driver-nvcc/index.html#x-c-c-cu-x)
  - specifying cu is most likely going to be the safe bet, as that allows c++ and hip/cuda code, whereas c++ does not allow hip/cuda code afaik

Reply:
agree that --xcu seems to be viable solution

Reply:
Just did some testing, I confirm that we can pass `-x cu` or `--x cu` to nvcc through hipcc and nvcc is able to compile the .hip program.

Eg. This works: `HIP_PLATFORM=nvidia hipcc hello.hip -o hello -x cu`

Also, a sidenote: `-xcu` and `--xcu` are not parsed correctly by nvcc and result in 
`nvcc fatal   : Unknown option '[-]-xcu'`. This is a bit different than hipcc to amdclang++ which can parse `-xhip` properly by splitting the single character x flag from its argument. 

I will amend this doc change and clarify that it pertains to the HIP_PLATFORM=nvidia hipcc usecase.